### PR TITLE
Flutter freezed docs

### PR DESCRIPTION
**Pull request by Parables**
## Description

Added documentation for the `flutter-freezed` plugin
 
Related # (issue)

## Type of change

- [x ] This change requires a documentation update

## Checklist:

- [x ] I have followed the [CONTRIBUTING](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md) doc and the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ x] I have added tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass locally with my changes
- [ x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Parables** [comment]
@charlypoly  Please let me know if there is anything else I need to do

File: website/src/category-to-packages.mjs [hunk 1]
```diff
@@ -46,5 +46,5 @@ export const CategoryToPackages = {
     'typescript-vue-apollo-smart-ops',
     'typescript-vue-urql',
   ],
-  dart: ['flutter'],
+  dart: ['flutter-freezed'],
```
Comment (charlypoly) [inline website/src/category-to-packages.mjs:49]:
```suggestion
  dart: ['flutter'],
```

**Review by charlypoly — APPROVED** [review]

**Comment by Parables** [comment]
@charlypoly, thanks for merging the docs. Sincerely grateful.

Howeverr, there are some typos I need to fix and send in another PR so I will do that and get back to you before 5:00am UTC/GMT

**Comment by Parables** [comment]
@charlypoly  
PS: the FlutterFreezedPluginConfig API docs for the plugin is not showing:

How do I fix it or can you look into it for me. 

Thanks once again

**Pull request merged**